autoload/lsp/ui/vim/signature_help.vim

```diff
@@ -51,6 +83,30 @@ function! s:handle_signature_help(server, data) abort
     endif
 endfunction
 
+function! s:get_parameter_label(signature, parameter) abort
+    if has_key(a:parameter, 'label')
+        if type(a:parameter) == v:t_list
+            let l:string_range = a:parameter['label']
+            return strcharpart(l:signature['label'], l:string_range[0], l:string_range[1])
+        endif
+        return a:parameter['label']
+    endif
+    return ''
+endfunction
+
+function! s:get_parameter_doc(parameter) abort
+    if !has_key(a:parameter, 'documentation')
+        return ''
+    endif
+
+    if type(a:parameter['documentation']) == v:t_dict
+        let l:doc = copy(a:parameter['documentation'])
+        let l:doc['value'] = printf('***%s*** - %s', a:parameter['label'], l:doc['value'])
```

Comment:
Should I check kind is `markdown`?

Reply:
This should work for both cases, I believe, so a check is not necessary.